src/mmw/js/src/core/models.js

```diff
@@ -60,14 +60,26 @@ var MapModel = Backbone.Model.extend({
         this.set('areaOfInterest', _.clone(this.get('previousAreaOfInterest')));
     },
 
-    setHalfSize: function(fit) {
-        this.set('size', { half: true, fit: !!fit });
+    setDoubleSmallSize: function(fit) {
+        this._setSizeOptions({ double: true  }, { small: true }, fit);
     },
 
-    setFullSize: function(fit) {
-        this.set('size', { half: false, fit: !!fit });
+    setDoubleHalfSize: function(fit) {
```

Comment:
Consider renaming these to something like `setDoubleHeaderSmallFooterSize' and `setDoubleHeaderHalfFooterSize'.